core/sql/comexe/CmpMessage.h

```diff
@@ -419,8 +415,7 @@ class CmpCompileInfo
 
   char * sqltext_;    // 00-03
   Lng32 sqlTextLen_;   // 04-07
-  RecompLateNameInfoList * rlnil_;  // 08-11
-  Lng32 rlnilLen_;  // 12-15
+  Lng32 unused_;       // 12-15
```

Comment:
Should this be an Int64? and should the comment say, // 08-15? (Though I don't think it matters anymore since we don't do static SQL.)